src/intree.js

```diff
@@ -22,6 +22,11 @@ function completeInTreeConfig(config, payload) {
     config.scopes = [
       prefix + payload.details['event.base.repo.branch'],
     ];
+  } else if (payload.details['event.type'] == 'release') {
+    let prefix = `assume:repo:github.com/${ payload.organization }/${ payload.repository }:branch:`;
+    config.scopes = [
+      prefix + payload.details['event.base.repo.branch'] + ':release',
```

Comment:
I sorta wonder whether or not we care what branch the release came from? I don't really understand releases though, so if this seems right (and if it seems right to bhearsum) this is fine.